### PR TITLE
optimalization

### PR DESCRIPTION
**Pull request by marwin1991**
This branch:(check if web_dev is from feature/package branch)
https://developers.google.com/speed/pagespeed/insights/?hl=pl&url=https%3A%2F%2Flibamtrack.github.io%2Fweb_dev%2F&tab=desktop

From master:
https://developers.google.com/speed/pagespeed/insights/?hl=pl&url=https%3A%2F%2Flibamtrack.github.io%2Fweb%2F&tab=desktop

### CODE REVIEW

**Comment by marwin1991** [comment]
Can you test if site works? I've check a few functions and it looks okay.

File: webpack.config.js [hunk 1]
```diff
@@ -53,5 +57,8 @@ module.exports = {
             },
         ]
     },
-    plugins: [htmlPlugin, copyStatic, copyWasm]
-};
+    plugins: [htmlPlugin, copyStatic, copyWasm, new BundleAnalyzerPlugin({"openAnalyzer":false, "analyzerMode":"static"}),
+        new webpack.DefinePlugin({
+        'process.env.NODE_ENV': JSON.stringify('production')
+    }), new Visualizer({"filename":"report.html"})]
```
Comment (grzanka) [inline webpack.config.js]: what is the purpose of `report.html` ?

Comment (marwin1991) [reply webpack.config.js]: To control bundle size during development, you will be able to visit it and see bundle visualization. This helped me notice that plotly.js is really big and that is why we use smaller verison which don't include some bussines charts etc

Comment (grzanka) [reply webpack.config.js]: Got it, this PR can be merged.

**Comment by grzanka** [comment]
@marwin1991 what is the status of this PR ? can we merge it ?

**Comment by marwin1991** [comment]
I will remove conflic and merge it

**Pull request merged**